Update documentation for workstation keys (#2419)
chef/chef-server: docs-chef-io/content/server/auth.md

#### Workstations

Each workstation stores its private key in the user's `~/.chef` directory.
This private key is generated by the Chef Infra Server and must be download

> from the server and copied to the `~/.chef` directory manually. If a new
> private key is required, it can be regenerated from the Chef Infra Server
> and copied to the `~/.chef` directory again.

Reviewer: ```suggestion
from the server and copied to the `~/.chef` directory manually. If you 
require a new private key, generate it with the Chef Infra Server and 
copy it to the `~/.chef` directory again.
```
